tests/test_bug_features.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import os
+
+from bugbug.bug_features import component
+from bugbug.bug_features import has_crash_signature
+from bugbug.bug_features import has_github_url
+from bugbug.bug_features import has_regression_range
+from bugbug.bug_features import has_str
+from bugbug.bug_features import has_url
+from bugbug.bug_features import has_w3c_url
+from bugbug.bug_features import is_coverity_issue
+from bugbug.bug_features import keywords
+from bugbug.bug_features import landings
+from bugbug.bug_features import patches
+from bugbug.bug_features import product
+from bugbug.bug_features import severity
+from bugbug.bug_features import title
+from bugbug.bug_features import whiteboard
+
+
+def read(filename, cls, expected_results):
+    path = 'tests/fixtures/bug_features/' + filename
+    feature_extractor = cls()
+    assert os.path.exists(path)
+
+    data = []
+    with open(path, 'r') as f:
+        for line in f:
+            data.append(feature_extractor(json.loads(line)))
```

Comment:
Nit: you could assert here directly, doing something like:
```Python
results = (feature_extractor(json.loads(line)) for line in f)
for result, expected_result in zip(results, expected_results):
    assert result == expected_result
```

This way you will avoid building the `data` list, and you'll get a nicer message in case of errors.